Sources/XcodeGraph/Models/Plist.swift

```diff
@@ -123,7 +126,11 @@ public enum InfoPlist: Equatable, Codable, Sendable {
 
 extension InfoPlist: ExpressibleByStringLiteral {
     public init(stringLiteral value: String) {
-        self = .file(path: try! AbsolutePath(validating: value)) // swiftlint:disable:this force_try
+        if value.hasPrefix("$(") || value.hasPrefix("${") {
+            self = .variable(value)
+        } else {
+            self = .file(path: try! AbsolutePath(validating: value)) // swiftlint:disable:this force_try
+        }
     }
```

Comment:
What about using regex to ensure that the variables have the proper format?  _(i.e., not just the prefix)_

```
public init(stringLiteral value: String) {
    let regexPattern = #"^\$\((.+)\)$|^\$\{(.+)\}$"#
    if let _ = value.range(of: regexPattern, options: .regularExpression) {
        self = .variable(value)
    } else {
        self = .file(path: try! AbsolutePath(validating: value)) // swiftlint:disable:this force_try
    }
}
```

Reply:
Thank you for suggestion, done.